threeML/plugins/OGIPLike.py

```diff
@@ -118,7 +118,10 @@ def _output(self):
 
         superout = super(OGIPLike, self)._output()
 
-        this_out = {'pha file': self._observed_spectrum.filename, 'bak file': self._background_spectrum.filename}
+        this_out = {
+                    'pha file': self._observed_spectrum.filename,
+                    'bak file': self._background_spectrum.filename if self._background_spectrum is not None else None
+                    }
 
```

Comment:
Could you change the nested if statement to such that it reads like this:
```python

 if self._background_spectrum is not None:
    bak_file =  self._background_spectrum.filename

else:
    bak_file = None


this_out = {
                     'pha file': self._observed_spectrum.filename,
                     'bak file': bak_file

```
It is a bit easier to read. Ideally, I should make a dummy background spectrum in the future. Perhaps make similar changes throughout. Thanks for catching this!